### ISSUE TITLE
Definitions for Intl.RelativeTimeFormat missing

### ISSUE THREAD

**Issue by dilyanpalauzov** [microsoft/TypeScript#38034]
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RelativeTimeFormat/formatToParts says:

```javascript
const rtf1 = new Intl.RelativeTimeFormat('en', { numeric: 'auto' });
```
is valid, but TypeScript 3.8.3 does not know out-of-the box about RelativeTimeFormat.

**Comment by hediet** [comment]
See issue #29129 and the (closed?) pull request #36084.

**Comment by dilyanpalauzov** [comment]
TypeScript does not know about Intl.RelativeTimeFormat.

https://github.com/microsoft/TypeScript/pull/36084 was discarded, as there was no activity on it and https://github.com/microsoft/TypeScript/issues/29129 was about an ECMAScript-Draft at the time.
